Framework/DataObjects/test/MDGridBoxTest.h

```diff
@@ -792,8 +792,8 @@ class MDGridBoxTest : public CxxTest::TestSuite {
           MDEventsTestHelper::makeRecursiveMDGridBox<2>(numSplit,
                                                         recurseLevels);
 
-      for (double x = 0; x < numSplit; x += spacing)
-        for (double y = 0; y < numSplit; y += spacing) {
+      for (double x = 0; x < static_cast<double>(numSplit); x += spacing)
+        for (double y = 0; y < static_cast<double>(numSplit); y += spacing) {
```

Comment:
In general I don't like [floating-point variables as loop counters](https://www.securecoding.cert.org/confluence/display/c/FLP30-C.+Do+not+use+floating-point+variables+as+loop+counters). That said, I don't see any benefit to making larger changes in a disabled test. 👍